MmSupervisorPkg/Test/MmSupvRequestUnitTestApp/MmSupvMemPolicyUnitTest.c

```diff
@@ -422,6 +422,7 @@ VerifyMemPolicy (
             // Shoot, found an overlap and we are an allow list...
             if ((MemoryMap->Attribute & EFI_MEMORY_XP) == 0) {
               // Check the paging attribute to see if this really is a code page
+              DEBUG ((DEBUG_INFO, "Memory map contains a bad memory type: %x at Index1: %d, Index2: %d! \n", MemoryMap->Type, Index1, Index2));

```

Comment:
How are the print error levels decided? Is there a reason to only print this at `DEBUG_INFO`?

Reply:
I also suggest prefixing hex numbers with `0x` to ease interpretation.

Reply:
I changed most of the messages to DEBUG_ERROR.  I left a few as DEBUG_INFO to match the final level print.